### ISSUE TITLE
Can't use styles in WYSIWYG text editor

### ISSUE THREAD

**Issue by lucacattide** [strapi/strapi#4165]
**Describe the bug**
Applying any style on a text inside the **text editor**, generates this error at run-time on a NextJS app:

> Error while running `getDataFromTree` { Invariant Violation: The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.

Then the page doesn't be rendered. Actually I'm working on ```localhost``` with _Apollo_ & _GraphQL_

**Steps to reproduce the behavior**
1. Create a content type with a text-type field
2. Create an associated content, populate the editor with some styled text (ex. bold, italic, etc) and save it.
3. Try to run the related front-end page

**Expected behavior**
The app should be normally rendered with its styled content correctly displayed

**Screenshots**
![Screenshot 2019-10-02 at 12 05 20](https://user-images.githubusercontent.com/12085296/66036242-0f90e380-e50d-11e9-899a-09157dd82d52.png)
![Screenshot 2019-10-02 at 12 05 52](https://user-images.githubusercontent.com/12085296/66036243-10297a00-e50d-11e9-9914-63bcdf9efc21.png)
![Screenshot 2019-10-02 at 12 06 39](https://user-images.githubusercontent.com/12085296/66036266-1c153c00-e50d-11e9-83e1-5e2980a671b8.png)

**Code snippets**
This is the line in which contents should be rendered:
```<h1 className="headline-home__claim occult">{props.headline}</h1>```

**System**
- Node.js version:  v10.16.0
- NPM version: 6.11.3
- Strapi version: v3.0.0-beta.15
- Database: MongoDB (localhost)
- Operating system: Mac OS X 10.14.6

**Additional context**
Basically I can't use styles? Am I missing something or it will be fixed on next releases?

Thanks in advance for your help

**Comment by lucacattide** [comment]
Apologize for this issue, just found on my code that there was this attribute

```style="border-width:0"```

instead of

```style={{ borderWidth: 0 }}```

so the error was caused by it.
But what about the style render? Now, my text is shown like ```***string***``` instead of **string**. The contents are being rendered as shown in the editing side - as you can see from my screens above

**Comment by soupette** [comment]
@lucacattide the strapi's WYSIWYG stores markdown in your database so if you need to render in html you can either use a component like [react-markdown](https://github.com/rexxars/react-markdown) or a converter like [showdown](https://github.com/showdownjs/showdown)

**Comment by lucacattide** [comment]
Thanks @soupette for your suggestions. I'll try them.
